app/models/beta/search/open_search_result.rb

```diff
@@ -137,6 +137,17 @@ def redirect!
       def redirect?
         @redirect
       end
+
+      def intercept_message
+        searched_term = search_query_parser_result.corrected_search_query
+
+        return unless searched_term.eql?((I18n.t "#{searched_term}.title"))
+
+        {
+          term: (I18n.t "#{searched_term}.title"),
```

Comment:
I don't think you need to change the evaluation rules with these extra (). Essentially `I18n.t "#{searched_term}.title"` will return the thing you want.